src/common/utils.js

```diff
@@ -0,0 +1,16 @@
+export const getIsMobile = () => {
```

Comment:
Another question, any reference where this was taken from ?
And how does this work ?
_My assumption is looking and parsing the request headers ?_

Reply:
It checks against the user agent: https://stackoverflow.com/questions/11381673/detecting-a-mobile-browser